### ISSUE TITLE
Deep linking using protocol handlers

### ISSUE THREAD

**Issue by dchapkine** [trueinteractions/tint2#148]
Is it possible to register protocol handlers with tint ?  Like itunes/appstore does for example.

Usage example:
- User receives an email with some link. ex: myapp://something
- When he/she clicks it, it calls   myapp:// handler if the app is installed on the system, then the app does some custom action

**Comment by trevorlinton** [comment]
@dchapkine We have this added for OSX, unfortunately its proving a bit more difficult to get it cross-platform compatible and stable. We'll have a delayed release this week and this will be added for the release on July 20th (next monday).

**Comment by dchapkine** [comment]
@trevorlinton awesome news,
thanks !

**Comment by trevorlinton** [comment]
Added on https://github.com/trueinteractions/tint2/commit/0baeb479cd70240d488e401fb1ab4ff78facfa0f this will be available july 20th as part of the 2.1.19 release.

Please reopen if you have further questions.

**Comment by dchapkine** [comment]
Great, so it works in both windows and mac now ?


Is it supposed to be used like this ?

```

function openWithMyApp(urls) {
	
		console.log("files or url to handle", urls);
};


if (application.isRegisteredForFileType('myfiletype') || application.registerFileType('myfiletype'))
{
	application.addEventListener('open', openWithMyApp);
}

if (application.isRegisteredForScheme('myscheme') || application.registerScheme('myscheme'))
{
	application.addEventListener('open', openWithMyApp);
}

```



After registering filetype, will the app automatically appear in "open with" menu for files with "myfiletype" extension ?



In cli mode like this ?

```
myapp --open-file filename filename

```



Does this requires an entitlement for a sandboxed OSX app ?

From my understanding while reading [this](https://developer.apple.com/library/mac/documentation/Miscellaneous/Reference/EntitlementKeyReference/Chapters/EnablingAppSandbox.html#//apple_ref/doc/uid/TP40011195-CH4-SW6) it does work using "com.apple.security.files.user-selected.read-only", what do you think about this ?
